avocado/utils/linux_modules.py

```diff
@@ -96,7 +101,23 @@ def loaded_module_info(module_name):
     :rtype: dict
     """
     l_raw = process.system_output('/sbin/lsmod')
-    return parse_lsmod_for_module(l_raw, module_name)
+    modinfo_dic = parse_lsmod_for_module(l_raw, module_name)
+    output = process.system_output("/sbin/modinfo %s" % module_name)
+    if output:
+        param_list = []
+        for line in output.splitlines():
+            key = line.split()[0]
+            if key == 'filename:':
+                modinfo_dic['filename'] = line.split()[-1]
+            elif key == 'version:':
+                modinfo_dic['version'] = str(line.split()[-1])
+            elif key == 'depends:':
+                if len(line.split()) > 1:
+                    modinfo_dic['depends'] = line.split()[1].split(',')
+            elif key == 'parm:':
+                param_list.append(line.split()[1].split(':')[0])
```

Comment:
In previous PR I asked about values and accordingly to response I think they want to just get the list of them and if needed they'd implement function to get value/type/... of the each parameter rather than populating this generic dict with many details. But I might be wrong.